### ISSUE TITLE
New services are unkown since Icinga 2 restart.

### ISSUE THREAD

**Issue by mphilipps** [Icinga/icingaweb2#4060]
hi,
This might be an icinga2 web bug, in that case please move this issue.

## Describe the bug

Yesterday I have added a new host+services through editing the config and reloading. Afterwards I noticed that one of the new services is consider to be in the state unknown since the end of last year. This also happens to be the last time the Icinga2 service was restarted.
## To Reproduce

1. restart Icinga
2. wait  a few minutes
3. create a new service that is expected to enter the unknown state
4. check the "unknown since" field in icinga2 web

## Expected behavior

The unknown since field should display since when the service changed from pending to unknown or if that isn't possible since the service object was created.

## Screenshots

![screen](https://user-images.githubusercontent.com/17722151/72788177-abb43700-3c31-11ea-8dc4-b0c432efa5c3.png)

## Your Environment

Include as many relevant details about the environment you experienced the problem in

* Version used (`icinga2 --version`): r2.11.2-1
* Operating System and version: Debian GNU/Linux 9.11 (stretch)
* Enabled features (`icinga2 feature list`): api checker command ido-pgsql livestatus mainlog notification
* Icinga Web 2 version and modules (System - About): 2.7.3

**Comment by nilmerg** [comment]
Hi,

I don't quite get the issue you have. Isn't it shown since when it's unknown? (2019-12) Do you expect the full date and time? (Just hover over it, then it should be displayed) In the detail area it's also shown in full.

**Comment by mphilipps** [comment]
hi,
It does, however the server and service in question hasn't existed in 2019. Yesterday I installed the server, there can't be service for it that is unknown since 2019.

**Comment by nilmerg** [comment]
I see. Though, it's Icinga which is using the time and date of the last launch as last update time of a new host/service. We can't differentiate this from a real update time so unless this changes in Icinga we can't fix it here.

But there's already https://github.com/Icinga/icinga2/issues/7784 due to a different requirement. This might have also an effect on the IDO so there's hope. :crossed_fingers:

**Comment by mphilipps** [comment]
Yes, I wasn't sure whether this is an Icinga2 or Icingaweb issue. That is why I created the ticket in the icinga2 tracker.

**Comment by nilmerg** [comment]
Soooo, took a look at it now. It's clearly an Icinga 2 issue. A host/service that enters the _UNKNOWN_ state upon the very first check gets the time and date of Icinga 2's launch as last state change. By restarting Icinga 2 after changing the configuration this isn't as obvious as when reloading Icinga 2.
